packages/orbit-tailwind-preset/src/presets/foundation/__tests__/createCustomFoundation.test.ts

```diff
@@ -0,0 +1,177 @@
+import { defaultFoundation } from "@kiwicom/orbit-design-tokens";
+
+import { createCustomFoundation } from "../createCustomFoundation";
+
+describe("createCustomFoundation", () => {
```

Comment:
I don't know what all these tests and functions are for 😕 they all seem rather extra to what the scope of this MR is

Reply:
Partially yes, it can be split on two MRs, but the logic inside will have to be still adjusted to make it themeable. The difference is whether the config will receive an additional parameter or not. (this can be abstracted to separated MR). 

The current problem -> when we define the palette with css variables, it won't have any effect on other tokens that are related to components, because they are now defined with `defaultTokens.<token>`. So the logic has to be adjusted to use `getTokens(cssVarsFoundation)` and passed down to both presets, so all the tokens will inherit the correct foundation. So that's why the PR eventually is bigger than it was expected, it's not just rewriting the values of the palette to css variables. Plus I am removing in this PR the things that are imo overengineered like this [one](https://github.com/kiwicom/orbit/blob/master/packages/orbit-tailwind-preset/src/presets/foundation/helpers.ts#L64C14-L64C34). I understand that it's a lot of change at once, so I will try to split it, but we have to also agree, that sometimes things can be significantly smaller as the problem is not that easy as it might look :)